kafka_acl_enforcer/src/main/java/com/tesla/data/acl/mixin/AclBindingMixin.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020. Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.acl.mixin;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.kafka.common.acl.AccessControlEntry;
+import org.apache.kafka.common.resource.ResourcePattern;
+
+public abstract class AclBindingMixin {
```

Comment:
Not totally obvious how the mixins work with mapping back to the real objects, maybe a package-info.java to explain? Or a couple comments and a link back to the appropriate jackson docs.